src/Command/SyncRecipesCommand.php

```diff
@@ -80,6 +88,45 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $operations[] = new InstallOperation($pkg);
         }
 
+        if ($createEnvLocal = $force && file_exists($this->rootDir.'/.env') && file_exists($this->rootDir.'/.env.dist') && !file_exists($this->rootDir.'/.env.local')) {
+            $io->writeError('Renaming .env -> .env.local and .env.dist -> .env');
+            rename($this->rootDir.'/.env', $this->rootDir.'/.env.local');
+            $pipes = [];
+            proc_close(proc_open(sprintf('git mv .env.dist .env > %s 2>&1 || %s .env.dist .env', $win ? 'NUL' : '/dev/null', $win ? 'rename' : 'mv'), $pipes, $pipes, $this->rootDir));
+            if (file_exists($this->rootDir.'/phpunit.xml.dist')) {
+                touch($this->rootDir.'/.env.test');
```

Comment:
Nothing written on it?

Reply:
Nope. The command will populate it.

Reply:
Great, I wasn't sure of this point, thought it could've been nice to copy/paste env to this file, but if it's automatic it's great 👍